solr/core/src/java/org/apache/solr/cli/DeleteTool.java

```diff
@@ -122,16 +124,36 @@ public List<Option> getOptions() {
             .desc(
                 "Skip safety checks when deleting the configuration directory used by a collection.")
             .build(),
+        Option.builder("p")
+            .longOpt("port")
+            .deprecated(
+                DeprecatedAttributes.builder()
+                    .setForRemoval(true)
+                    .setSince("9.7")
+                    .setDescription("Use --solr-url instead")
+                    .get())
+            .argName("PORT")
+            .hasArg()
+            .required(false)
+            .desc("Port of a local Solr instance where you want to create the new core.")
+            .build(),
         SolrCLI.OPTION_SOLRURL,
         SolrCLI.OPTION_SOLRURL_DEPRECATED,
         SolrCLI.OPTION_ZKHOST,
-        SolrCLI.OPTION_ZKHOST_DEPRECATED);
+        SolrCLI.OPTION_ZKHOST_DEPRECATED,
+        SolrCLI.OPTION_VERBOSE_DEPRECATED);
```

Comment:
Should this be eventually
Same here:
```suggestion
        SolrCLI.OPTION_ZKHOST_DEPRECATED,
        SolrCLI.OPTION_VERBOSE,
        SolrCLI.OPTION_VERBOSE_DEPRECATED);
```